tutorials/metrics.md

```diff
@@ -0,0 +1,39 @@
+# CRI-O Metrics
+
+To enable the [Prometheus][0] metrics exporter for CRI-O, either start `crio`
+with `--metrics-enable` or set the corresponding option in
+`/etc/crio/crio.conf`:
+
+```toml
+[crio.metrics]
+enable_metrics = true
+```
+
+The metrics endpoint serves per default on port `9090`. This can be changed via
+the `--metrics-port` command line argument or via the configuration file, too:
+
+```toml
+metrics_port = 9090
+```
+
+If CRI-O runs with enabled metrics, then this can be verified by querying the
+endpoint manually via [curl][1].
+
+```bash
+> curl localhost:9090/metrics
+…
+```
+
+## Available Metrics
+
+Beside the [default golang based metrics][2], CRI-O provides the following additional metrics too:
```

Comment:
I'd drop 'too' here also.    Just end with 'metrics:'

Reply:
Yep, fixed :+1:

Reply:
'too:' looks to have survived....

Reply:
Ah, fixed (finally)